static/local-authentication/styles/main.css

```
@@ -0,0 +1,83 @@
+  font-family: system-ui;
+  font-size: 16px;
+}
+
+html,
+body,
+canvas {
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  user-select: none;
+  touch-action: none;
+  user-select: none;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

user-select is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)